rpc_util.go

```diff
@@ -365,6 +366,36 @@ func (o MaxRecvMsgSizeCallOption) before(c *callInfo) error {
 }
 func (o MaxRecvMsgSizeCallOption) after(*callInfo, *csAttempt) {}
 
+// CallAuthority returns a CallOption which sets the authority to override the
+// `:authority` pseudoheader in a RPC. If this is set, the RPC will use only
+// this authority.
+//
+// # Experimental
+//
+// Notice: This API is EXPERIMENTAL and may be changed or removed in a
+// later release.
+func CallAuthority(auth string) CallOption {
```

Comment:
Nit: s/auth/authority/

Please note that this is part of our public API. Parameter names also serve as documentation. So using good parameter names is critical.